NumberBaseball/NumberBaseball/main.swift

```diff
@@ -7,34 +7,126 @@
 import Foundation
 
 let winnerGoalCount = 3
+
+let numberRange: ClosedRange<Int> = 1...9
+let limitCount = 3
+
 var strikeCount = 0
 var ballCount = 0
 
+func startProgram() {
+    printMenu()
+    runSelectedMenu(menuInput: selectedMenu())
+}
+
+func printMenu() {
+    print("1. 게임시작")
+    print("2. 게임종료")
+    print("원하는 기능을 선택해주세요 : ", terminator: "")
+}
+
+func selectedMenu() -> Int {
+    guard let menuInput = readLine(),
+          let convertedMenuInput = Int(menuInput)
+    else {
+        return .zero
+    }
```

Comment:
```suggestion
guard let menuInput = readLine(),
      let convertedMenuInput = Int(menuInput) else {
    return .zero
}
```
전 이렇게 코드를 작성합니다. 어떤게 더 가독성이  좋아보이시나요? 안고치셔도 됩니다 ㅎㅎ

Reply:
얘기해봤더니 지성 방식이 더 보기 편해보여서 바꿨어요!